<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeDialog</class>
 <widget class="QDialog" name="HomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎使用</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RC.qrc">
    <normaloff>:/new/prefix1/rc/Home.ico</normaloff>:/new/prefix1/rc/Home.ico</iconset>
  </property>
  <widget class="QGroupBox" name="HomeGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>欢迎使用Forvo真人语音下载器</string>
   </property>
   <widget class="QRadioButton" name="NewRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>创建新的单词下载列表</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="OldRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>使用现有单词下载列表</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OldFileLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenOldButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
    <property name="icon">
     <iconset resource="RC.qrc">
      <normaloff>:/new/prefix1/rc/Folder3 (2).ico</normaloff>:/new/prefix1/rc/Folder3 (2).ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
    <property name="icon">
     <iconset resource="RC.qrc">
      <normaloff>:/new/prefix1/rc/Forbidden (2).ico</normaloff>:/new/prefix1/rc/Forbidden (2).ico</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="icon">
    <iconset resource="RC.qrc">
     <normaloff>:/new/prefix1/rc/Ok (2).ico</normaloff>:/new/prefix1/rc/Ok (2).ico</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="RC.qrc"/>
 </resources>
 <connections/>
</ui>
